src/main/java/io/fabric8/jenkins/openshiftsync/Constants.java

```diff
@@ -45,6 +45,7 @@
     public static final String OPENSHIFT_SECRETS_DATA_SSHPRIVATEKEY = "ssh-privatekey";
     public static final String OPENSHIFT_SECRETS_DATA_FILENAME = "filename";
     public static final String OPENSHIFT_SECRETS_DATA_CERTIFICATE = "certificate";
+    public static final String OPENSHIFT_SECRETS_DATA_SECRET_TEXT = "secrettext";
```

Comment:
Open to suggestions of a more intuitive key name. Maybe `token`? Or maybe it doesn't really matter

Reply:
I'm good with what you have here